elk/extensions/recursive_ccs/train.py

```diff
@@ -0,0 +1,110 @@
+import csv
+import pickle
+import random
+
+import numpy as np
+import torch
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score, roc_auc_score
+from tqdm.auto import tqdm
+
+from elk.files import elk_cache_dir
+from elk.training.ccs import CCS
+from elk.training.preprocessing import load_hidden_states, normalize
+from elk.extensions.recursive_ccs.rccs import RecursiveCCS
+from elk.extensions.recursive_ccs.parser import get_rccs_training_parser
+
+
+@torch.autocast("cuda", enabled=False)
```

Comment:
Should just be removed; there's no outer context that's turning `autocast` on so we don't need to explicitly turn it off here

Reply:
Done

Reply:
Also, torch's documentation advises against using autocast for training. Maybe this should be disabled in the regular train.py?

Reply:
Um, that's just wrong? Autocast is frequently used for training. You can run into problems if you don't use `GradScaler` though

Reply:
But we don't use GradScaler, right?

Reply:
Yep that's why I'm getting rid of autocast in the `semi-supervised` branch. LBFGS currently isn't compatible with `GradScaler` annoyingly. In the future I might try to manually add grad scaling to LBFGS